public/send-results.js

```diff
@@ -0,0 +1,26 @@
+/* global: location, sessionStorage */
+
+(function() {
+  var path = location.pathname.slice(1)
+
+  if (path === "login") {
+    var submitSurveyButton = document.getElementById('submit-survey-button')
+    var clinicianEmailInput = document.getElementById('clinician-email-input')
+    submitSurveyButton.addEventListener('click', function() {
+      var storeClinicianEmail = sessionStorage.setItem('clinician-email', clinicianEmailInput.value)
+    })
+  }
+
+  if (path === "results") {
+    var sendResultsButton = document.getElementById('send-results-button')
+    var getSessionStorage = window.sessionStorage
+    var getClinicianEmail = getSessionStorage['clinician-email']
+    var emailSubject = getSessionStorage['select-survey'] + ' results ' + Date()
+    var emailBody = JSON.stringify(getSessionStorage)
```

Comment:
window.sessionStorage allows me to access the entire sessionStorage object. Otherwise I would have to specify all of the key value pairs that I want, for all the answers for both surveys. 
The syntax is the same for localStorage, as far as I know: https://developer.mozilla.org/en-US/docs/Web/API/Window/localStorage
An alternative is to have 2 conditions, with arrays for each set of questions, then you'd loop through and do getItem on each one. I did think about this. 
Let me know what you think 
Also something has broken so please don't merge  yet :p